Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp

```diff
@@ -241,7 +246,7 @@ class InternalSource final : public MediaStreamTrackPrivate::Observer,
         }
 
         if (m_track.isVideo() && drop)
-            drop = doCapsHaveType(caps, "video/x-raw") || GST_BUFFER_FLAG_IS_SET(buffer, GST_BUFFER_FLAG_DELTA_UNIT);
+            drop = doCapsHaveType(caps, "video") || GST_BUFFER_FLAG_IS_SET(buffer, GST_BUFFER_FLAG_DELTA_UNIT);
```

Comment:
if track isVideo, perhaps checking its caps -so generically- might be spurious.

Reply:
I'm not sure what you mean here, the caps can be either `video/x-raw` or `application/x-rtp media=video`.

Reply:
oh, ok! I didn't figured out the rtp side. Sorry for the noise.